modules/my_health/app/controllers/my_health/v1/folders_controller.rb

```diff
@@ -64,6 +65,10 @@ def create_folder_params
       def search_params
         params.permit(:exact_match, :sender, :subject, :category, :recipient, :from_date, :to_date, :message_id)
       end
+
+      def requires_oh_messages
+        params[:requires_oh_messages].try(:to_s) || '0'
+      end
```

Comment:
@mmoyer-va  good point. I think it may not be necessary to return `'0'` if param is not valid since the `append_requires_oh_messages_query` method in  client class would only do something if the value is equal to `'1'`. What do you suggest?